src/analysis/normed_space/riesz_lemma.lean

```diff
@@ -0,0 +1,30 @@
+import analysis.normed_space.basic
```

Comment:
You should include a header (with author and licence), and a file-level docstring explaining the contents of the file (it could be very short in this case, but in any case it would help to have it). See https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/doc.md for more details.

Reply:
Since the proof is short and not very difficult, my expectation had been that this should belong in some other file instead of on its own, only that I didn't know where.

I've put in a header comment nevertheless.